content/reference/current/plugins/github_actions/includes/use-case-binary-repo.sbt.md

```diff
@@ -0,0 +1,56 @@
+```yaml
+name: Run Gatling Enterprise Simulation
+
+# Execute the workflow on each push to the main branch
+on:
+  push:
+    branches:
+      - main
+
+# Here we use concurrency to cancel previous executions if they are still
+# ongoing. Useful to avoid running the same simulation several times
+# simultaneously if we push several code changes in a short time.
+# See https://docs.github.com/actions/using-jobs/using-concurrency.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      # Check out your GitHub repository.
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      # Set up Java and the build tools (including SBT).
+      # You can configure other versions of the JDK, as long as they are
+      # supported by your version of Gatling and by your build tool.
+      # See https://github.com/actions/setup-java/blob/main/README.md for options.
+      - name: Setup JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '17'
+          cache: 'sbt'
+
+      # Build, package, and publish your Gatling project 
+      - name: Build Gatling simulation
+        # In this example, the SBT build uses credentials from a file, as documented in:
+        # https://www.scala-sbt.org/1.x/docs/Publishing.html#Credentials
+        run: |
+          echo $MAVEN_CREDENTIALS > ~/.sbt/.credentials
+          ./gradlew publish
+        env:
+          # Retrieve credentials file content from a GitHub secret
+          # See https://docs.github.com/en/actions/security-guides/encrypted-secrets
+          MAVEN_CREDENTIALS: ${{ secrets.MAVEN_CREDENTIALS }}
```

Comment:
Weird naming? called `MAVEN_CREDENTIALS` but is under .sbt?

Reply:
Changed to `SBT_CREDENTIALS`.